.github/patch-initial-checklist.md

```diff
@@ -35,6 +35,7 @@ Each porter is responsible for testing the PRs that fall under the focus of thei
   * If one or more PRs are not testing as expected: ping the PR authors and the porter of the relevant team and ask them if the change is a release blocker or not (you can also ping the team lead if any of them is not available). In general, if it's not a regression or there is no product/marketing reason why that PR is a blocker, all other PRs should default to not being blockers.
     * If there are blockers: stop the release and ask the PR author to fix them. If the PR author is AFK, ping the porter of their team.
     * If some PRs are not testing as expected but they are not blockers: revert the relevant commits, remove the changes from testing steps and changelog, open an issue (or reopen the original one) and proceed with the release.
+    * If minor issues are discovered during the testing, each team is responsible for logging them in Github
```

Comment:
Nit-pic: Given that all other sentences end with a fullstop, this one should also end with one.